### ISSUE TITLE
[Docs] Examples contain some invalid HTML

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5#571]
```html
	<figure class="image illustration" style="height:400px;"><img alt="Picture of a farmer in a rice field." height="auto"
	                                                              src="https://c.cksource.com/a/1/img/demo/vietnam.jpg" width="100%">
```

or:

```html

	<ul>
		<li><a href="http://www.polin.pl/en" target="_blank" rel="external">POLIN</a></li>
		<li><a href="http://www.1944.pl/en" target="_blank" rel="external">Warsaw Uprising Museum</a></li>
		<li><a href="http://chopin.museum/en" target="_blank" rel="external">Fryderyk Chopin Museum</a></li>
	</ul>
```

By invalid, I mean invalid from CKEditor's perspective. Can I just clean this up?

**Comment by Reinmar** [comment]
cc @wwalc

**Comment by wwalc** [comment]
Yes, go ahead.

**Comment by Reinmar** [comment]
Closed with 2544173.
